### ISSUE TITLE
[COGNITION]: Missing error message for 526 forwarding address effective date inputs

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#2012]
## Description

When adding a forwarding address on the form 526 File for increased disability compensation address page - https://staging.va.gov/disability-benefits/apply/form-526-disability-claim/veteran-details/address-information - the effective date block is required, but does not show any error messages if left blank.

I noticed this issue after [adding an error message](https://github.com/department-of-veterans-affairs/vets-website/pull/10785/files#diff-3b52e4ccbb9b28eee6e2fedf67e27d8eR112-R114) in [PR #10785](https://github.com/department-of-veterans-affairs/vets-website/pull/10785); but it appears that the label not showing is an issue at the component level.

## Acceptance Criteria

- [ ] As a screen reader user, I want an error messages to alert me if I haven't entered an effective date for my forwarding address

## Environment

* Operating System: macOS Mojave v10.14.6
* Browser: Chrome, latest
* Screenreading device, if applicable
* Server destination: staging

## Steps to Recreate

1. Log in as user 226
2. Go to https://staging.va.gov/disability-benefits/apply/form-526-disability-claim/veteran-details/address-information
3. Click the continue button.
4. Check the "I want to provide a forwarding address since my address will be changing soon." check box.
5. Focus on and blur any of the "Effective date" inputs.
6. No error message is shown.

## Possible Fixes (optional)

In the effective date component, add the logic to display the error message. The reason the message may not be showing is because all other input blocks wrap their content in a `<label>` whereas the effective date component wraps the content in a `<legend>`.

## Screenshots or Trace Logs

<img width="564" alt="Screen Shot 2019-09-24 at 2 35 22 PM" src="https://user-images.githubusercontent.com/136959/65546461-316ee280-dedd-11e9-8806-54e9eefd66e1.png">

Proposed result

<img width="383" alt="Screen Shot 2019-09-24 at 3 13 45 PM" src="https://user-images.githubusercontent.com/136959/65546863-ef926c00-dedd-11e9-8f69-cdb3ca9be616.png">

**Comment by Mottie** [comment]
Strangely enough, I've started work on form 0996 (Higher-Level Review) which also needs this form field... While testing, I am seeing a message

![Screen Shot 2019-10-03 at 12 32 25 PM](https://user-images.githubusercontent.com/136959/66150108-1b96a700-e5da-11e9-9e61-66bea9fac9c0.png)

**Comment by Mottie** [comment]
So this issue can likely be closed.

The error message _is visible_ if the inputs are empty and after the "Continue" button on the form is pressed. It does not, however, show up when any of the three inputs are empty and lose focus.
